### PR TITLE
Move finding/creating organizer action needed list from rpc to route

### LINKED ISSUE #1203: Can't open view of blocked targets in new tab
## Description
A user should be able to click a link on a call assignment page, and be directed to a view containing all the people who are blocked from calling for some reason. This is currently possible, but not in a way that allows the user to cmd/ctrl+click the link to open it in a new browser window/tab.

## Steps to reproduce
1. Go to some call assignment, e.g. http://localhost:3000/organize/7/projects/standalone/callassignments/14
2. In the bottom left, locate the "View sheet" button (if you can't find it, you may need to fake some phone calls, see below) 
  a.  Go to http://call.dev.zetkin.org/assignments
  b. Sign in using `testadmin@example.com` and `password`
  c. Select the call assignment you looked at in step 1
  d. Go through the user flow to make some (fake) phone calls
  e. In at least one of the calls, when reporting it, answer yes to "did something happen that requires action by an official"
  f. Return to the call assignment on localhost 
3. Ctrl/cmd+click the button

### Expected Behaviour
Should open a view in a new tab

### Actual Behaviour
Opens in same tab

## Screenshots
This concerns the "View sheet" button in the bottom left of this screenshot.

image

## Proposed solution (refactor `getOrganizerActionView` RPC as redirect page instead)
Navigating to this view is currently accomplished using an RPC. When clicking the "View sheet" button on the call assignment page an RPC is invoked, which either finds an existing or creates a new view for this purpose, and returns its ID, at which point the router redirects to the view page.

This is where the UI for this exists:
https://github.com/zetkin/app.zetkin.org/blob/3b02385ce6c1f97cbce7450377a36cb116604d74/src/features/callAssignments/components/CallAssignmentStatusCards.tsx#L140-L152

And this is the model logic that does the redirect:
https://github.com/zetkin/app.zetkin.org/blob/3b02385ce6c1f97cbce7450377a36cb116604d74/src/features/views/models/ViewBrowserModel.ts#L167-L178

Because it's the router doing this redirect, it doesn't work as a link. Instead, I think the logic from the RPC should be moved over to a page route (in `src/pages`, e.g. `src/pages/organize/[orgId]/people/views/blocked.tsx`) in which the `getServerSideProps()` will do the work instead, and then redirect using a regular NEXT redirect.

### PR DESCRIPTION
**Pull request by ziggabyte**
## Description
This PR moves the logic that finds or creates a list for organizer action needed from an rpc to a route. 
The logic is not changed, just moved. 

## Screenshots
none

## Changes
* Removes getOrganizerActionView RPC
* Adds `organize/{orgId}/people/lists/blocked` route that does all that logic in its `getServersideProps` function.

## Related issues
Resolves #1203

### CODE REVIEW

File: src/features/callAssignments/components/CallAssignmentStatusCards.tsx [hunk 1]
```diff
@@ -150,13 +149,11 @@ const CallAssignmentStatusCards: FC<CallAssignmentStatusCardsProps> = ({
...
             <StatusCardItem
               action={
-                <Button
-                  onClick={() => getOrganizerActionView()}
-                  startIcon={<Visibility />}
-                  variant="outlined"
-                >
-                  <Msg id={messageIds.blocked.viewSheetButton} />
-                </Button>
+                <NextLink href={`/organize/${orgId}/people/lists/blocked`}>
```
Comment (richardolsson) [inline src/features/callAssignments/components/CallAssignmentStatusCards.tsx:152]: I'm thinking about what is a good URL for this list. The `blocked` name will be confusing once people can be blocked for more reasons (e.g. email), so I thin that part of the URL should change to `call_blocked`, or maybe `callblocked` ( to be more consistent with how `callassignments` are spelled in the URLs).

File: src/pages/organize/[orgId]/people/lists/blocked.tsx [hunk 2]
```diff
@@ -154,7 +146,22 @@ async function handle(
...
+      },
+    };
+  } else {
+    return {
+      notFound: true,
+    };
   }
+}, scaffoldOptions);
+
+export default function NotUsed(): null {
```
Comment (richardolsson) [inline src/pages/organize/[orgId]/people/lists/blocked.tsx:165]: Nice!

**Review by richardolsson — CHANGES_REQUESTED** [review]
I have reviewed the code and I have tried the preview build. Everything seems to work as it should and your solution looks generally good. I just have a comment about the path that I think should change to make room for the same type of feature but with email in the future.

File: src/pages/organize/[orgId]/people/lists/blocked.tsx (hunk 2 above)
Comment (ziggabyte) [reply src/pages/organize/[orgId]/people/lists/blocked.tsx:165]: It's stolen, from something you wrote 😂

File: src/features/callAssignments/components/CallAssignmentStatusCards.tsx (hunk 1 above)
Comment (ziggabyte) [reply src/features/callAssignments/components/CallAssignmentStatusCards.tsx:152]: will change to `callblocked`!

**Pull request merged**